service/templates/distros.html

```diff
@@ -517,6 +517,18 @@ <h2 style="display:inline-block; padding-right:15px">Configure
           <div class="icon fa fa-warning distro-error" ng-show="form.userName.$dirty && form.userName.$error.required || form.userName.$invalid">SSH
             user is required</div><br>
         </div>
+        <div class="dropdown">
+          <span class="distro-menu-title">Bootstrap Method:</span>
+            <button class="btn btn-default dropdown-toggle" type="button" data-toggle="dropdown" aria-expanded="true" ng-disabled="readOnly">
+              <strong class="distro-menu-item">[[ activeDistro.bootstrap_method | bootstrapMethodDisplay:this]]<span class="fa fa-caret-down"></span></strong>
+            </button>
+            <ul class="dropdown-menu" style="margin-left: 125px; align: left" role="menu">
+              <li required name="bootstrapMethodForm" ng-click="form.$setDirty();activeDistro.bootstrap_method = bootstrapMethod.id" ng-repeat="bootstrapMethod in bootstrapMethods"
```

Comment:
I'm not sure ng-click works on a li element. But this can probably be avoided by using a select

Reply:
See above.